pkg/asset/machines/master.go

```diff
@@ -407,6 +419,7 @@ func (m *Master) Machines() ([]machineapi.Machine, error) {
 	gcpapi.AddToScheme(scheme)
 	libvirtapi.AddToScheme(scheme)
 	openstackapi.AddToScheme(scheme)
+	ovirtprovider.AddToScheme(scheme)
 	decoder := serializer.NewCodecFactory(scheme).UniversalDecoder(
 		awsprovider.SchemeGroupVersion,
 		azureprovider.SchemeGroupVersion,
```

Comment:
probably need to add `ovirtprovider.SchemeGroupVersion` to decoder like all other platforms